small/small.c

```diff
@@ -162,27 +258,28 @@ smalloc(struct small_alloc *alloc, size_t size)
 {
 	small_collect_garbage(alloc);
 
-	struct factor_pool *upper_bound = factor_pool_search(alloc, size);
```

Comment:
Rename `upper_bound` -> `factor_pool` is a candidate for a journey to another commit.